src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Waiting.java

```diff
@@ -429,12 +343,12 @@ protected void waitUntil(String timestr, String message, WaitUntilFunction funct
 			}
 			try {
 				Thread.sleep(200);
-			} catch (InterruptedException e) {
+			} catch (InterruptedException ignored) {
 			}
 		}
 	}
 
-	protected static interface WaitUntilFunction {
+	protected interface WaitUntilFunction {
```

Comment:
This method look like a candidate for having [`FunctionalInterface` annotation](https://docs.oracle.com/javase/8/docs/api/java/lang/FunctionalInterface.html).

Reply:
Yep) fixed